backend/Application/Api/GraphQL/Import/AccountImportHandler.cs

```diff
@@ -21,11 +21,14 @@ public AccountImportHandler(IAccountLookup accountLookup, IMetrics metrics, Acco
         _writer = accountWriter;
     }
 
-    public async Task AddNewAccounts(AccountInfo[] createdAccounts, DateTimeOffset blockSlotTime)
+    public async Task AddNewAccounts(
+        AccountInfo[] createdAccounts, 
+        DateTimeOffset blockSlotTime, 
+        int blockHeight)
```

Comment:
Just to confirm, this is always "absolute block height" right? Not the "era" block height?

Reply:
Yes. Its the absolute block height.